src/lib/auto-complete/index.svelte

```diff
@@ -175,6 +179,9 @@
     );
     if (showListImmediately === true) {
       value = selectedWord;
+      if (selectedWord.endsWith("(")) {
+        value += ")";
```

Comment:
I think it would be preferable to update the the autocomplete values to include the the closing `)` instead of adding it here. It's unexpected to have the code here modify any of the autocomplete values when we're used to "seeing what you get" with the autocomplete value lists. But, It would add a bit more logic because you would have to check if the character before the cursor is a `)`.

Reply:
fixed